spec/features/user_can_read_email_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
```

Comment:
You don't have to require this file in every test file. It is required by default using the setting in `.rspec` in the project folder